shimmer/develop/explanations/what-is-stardust/dynamic-pow.md

```diff
@@ -0,0 +1,32 @@
+---
+keywords:
+  - Stardust
+  - Shimmer
+  - Upgrades
+  - Dynamic POW
+  - Proof-of-work
+  - Congestion control
+  - Protocol Improvements
+  - explanation
+description: Stardust prepares the protocol for the support of dynamic proof-of-work.
+image: /img/logo/preview.png
+---
+
+# Dynamic Proof of Work
+
+Proof of Work (PoW) is currently employed in IOTA for congestion control. Every block must include a small amount of
```

Comment:
Can we add the first line of this page up front? Seems weird to describe a non existing feature